### PR TITLE
FEAT: OmniSciDB supports round()

### PR DESCRIPTION
**Pull request by semelianova**

### CODE REVIEW

File: ci/requirements-3.6-dev.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ dependencies:
   - black=19.10b0
   - click
   - clickhouse-cityhash
-  - clickhouse-driver>=0.0.8,<0.1.2
+  - clickhouse-driver>=0.1.3
```
Comment (jreback) [inline ci/requirements-3.6-dev.yml]: revert these

Comment (semelianova) [reply ci/requirements-3.6-dev.yml]: Ok, it was done due to CI failure

Comment (jreback) [reply ci/requirements-3.6-dev.yml]: yes, let's merge #2089 first

**Comment by jreback** [comment]
pls merge master

**Comment by semelianova** [comment]
@xmnlab PR is ready for review

**Comment by jreback** [comment]
thanks @semelianova

File: docs/source/release.rst [hunk 2]
```diff
@@ -7,6 +7,7 @@ Release Notes
    These release notes are for versions of ibis **1.0 and later**. Release
    notes for pre-1.0 versions of ibis can be found at :doc:`/release-pre-1.0`
 
+* :support:`2096` Added round() support for OmniSciDB
```
Comment (xmnlab) [inline docs/source/release.rst]: I am wondering if it should be `feature` instead of `support` ...

**Pull request merged**